procrastinate/jobs.py

```diff
@@ -59,7 +59,10 @@ def get_context(self) -> types.JSONDict:
 
         if context["scheduled_at"]:
             context["scheduled_at"] = context["scheduled_at"].isoformat()
-
+        kwargs_string = ", ".join(
+            f"{key}={value}" for key, value in context["task_kwargs"].items()
+        )
+        context["call_string"] = f"{context['task_name']}({kwargs_string})"
```

Comment:
Can't we have sensitive information / PII in this? Another issue might be having long arguments, like something JSON encoded or a text.

I understand the value, but I wonder if it's not taking too much risks :) .

Reply:
> Can't we have sensitive information / PII in this?

There's nothing in there that won't also be in the extras ? Does it make a difference ?

> Another issue might be having long arguments, like something JSON encoded or a text.

That's right. Given the info will be present in the log, we could limit to, say 300 chars (configurable).

Reply:
(let's keep this PR simple and leave this out of scope)